docs/UserGuide.md

```diff
@@ -470,8 +468,6 @@ Important notes:
 > Location: John home;
 > Categories: #tuition
 > ```
->
-> <img src="images/unicash/command-outputs/editTransaction/editTransactionSuccessOutput2.png" width="1000" />
 
 ###### Example 3
```

Comment:
I think it's good to replace these with bolded titles instead so `**Example n**`, this way, the TOC isn't bloated with the headers